datalabeling/create_annotation_spec_set_test.py

```diff
@@ -17,7 +17,7 @@
 import os
 
 import backoff
-from google.api_core.exceptions import DeadlineExceeded
+from google.api_core.exceptions import ServerError
```

Comment:
Do we have any sense of how common these are? It seems bad for the service to be regularly returning `ServerError`s.

Reply:
I only saw once. I don't know the exact frequency, but it sounds ok to retring for all the 5xx errors anyways. WDYT?

Reply:
I mean, yeah it is important to track the frequency of 500 errors for sure, but it's the product teams' responsibility.